### DISCUSSION TITLE
Not able to access the Xpra Using Domain

### DISCUSSION THREAD

**Discussion by prajapatiraj-sudo** [Xpra-org/xpra#3903] [General]
We have taken VPS and this VPS has assigned a local IP Using Public ip it's Forwded to this server,

At that time while im login then it's Show the Error 

**how can i access the VPS using Domain and VPS Hase Local IP** 

![screen73](https://github.com/Xpra-org/xpra/assets/111242511/5e698c66-4f85-4269-b04b-7ecba738f18b)
![new](https://github.com/Xpra-org/xpra/assets/111242511/3c512e71-c8c8-4c49-8a5f-4763e3bd69d8)

**Comment by totaam** [comment]
There is nowhere near enough information to be able to diagnose anything.
Please see https://github.com/Xpra-org/xpra/wiki/Reporting-Bugs
